js/block-editor.js

```diff
@@ -11,7 +11,7 @@ wp.apiFetch.use(
 	function( options, next ) {
 		// If options.url is defined, this is not a REST request but a direct call to post.php for legacy metaboxes.
 		if ( 'undefined' === typeof options.url ) {
-			if ( 'undefined' === typeof options.data ) {
+			if ( 'undefined' === typeof options.data || null === options.data ) {
 				// GET
 				options.path += ( ( options.path.indexOf( '?' ) >= 0 ) ? '&lang=' : '?lang=' ) + getCurrentLanguage();
```

Comment:
Shouldn't we create a `getRequestType()` function that we could use in both Polylang and Polylang Pro, so we will not forget to update both if it change in the future?

Reply:
🤔 it isn't the same JS file enqueued. This one is enqueued only in Polylang. Polylang use another one build with webpack as the metabox React part